Formula/stox.rb

```diff
@@ -0,0 +1,19 @@
+class Stox < Formula
+  desc "Swift command-line tool to view and export tickers from finviz.com"
+  homepage "https://github.com/xbladesub/stox"
+  url "https://github.com/xbladesub/stox/archive/1.0.1.tar.gz"
+  sha256 "f037220a0b810775dad6e4df77f335aa9e96ed10b94784ae8d2821a69c628e3b"
+  license "MIT"
+  
+  depends_on xcode: "12.4"
+  
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release", "-Xswiftc",
+                    "-cross-module-optimization", "--enable-test-discovery"
+    bin.install ".build/release/stox"
+  end
+  
+  test do
+    system bin/"stox", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.